config/config.js

```diff
@@ -0,0 +1,17 @@
+require("dotenv").config();
```

Comment:
@vnbrew tham khảo cái này tổ chức lại db và config của sequelize nha
https://victoronwuzor.medium.com/how-to-setup-sequelize-migration-in-a-nestjs-project-b4aec1f88612
Mọi code mình đều đặt trong src hoặc test chứ ko đặt ngoài 
Vì cái sequelize là lib bên ngoài nên nó ko biết đặt đâu nhưng mình dùng nest thì phải đặt cho đúng vị trí

Reply:
@er-khanhtq đã sửa lại, chuyển vào folder src/database